src/Shared/StackTrace/StackFrame/PortablePdbReader.cs

```diff
@@ -17,111 +17,6 @@ internal class PortablePdbReader : IDisposable
-                var sequencePoints = methodDebugInfo.GetSequencePoints();
-                SequencePoint? bestPointSoFar = null;
-
-                foreach (var point in sequencePoints)
-                {
-                    if (point.Offset > IlOffset)
-                    {
-                        break;
-                    }
-
-                    if (point.StartLine != SequencePoint.HiddenLine)
-                    {
-                        bestPointSoFar = point;
-                    }
-                }
-
-                if (bestPointSoFar.HasValue)
-                {
-                    frameInfo.LineNumber = bestPointSoFar.Value.StartLine;
-                    frameInfo.FilePath = metadataReader.GetString(metadataReader.GetDocument(bestPointSoFar.Value.Document).Name);
-                }
-            }
-        }
-
-        private MetadataReader GetMetadataReader(string assemblyPath)
-        {
-            MetadataReaderProvider provider = null;
-            if (!_cache.TryGetValue(assemblyPath, out provider))
-            {
-                var pdbPath = GetPdbPath(assemblyPath);
-
-                if (!string.IsNullOrEmpty(pdbPath) && File.Exists(pdbPath) && IsPortable(pdbPath))
-                {
-                    var pdbStream = File.OpenRead(pdbPath);
-                    provider = MetadataReaderProvider.FromPortablePdbStream(pdbStream);
-                }
-
-                _cache[assemblyPath] = provider;
-            }
-
-            return provider?.GetMetadataReader();
-        }
-
-        private static string GetPdbPath(string assemblyPath)
-        {
-            if (string.IsNullOrEmpty(assemblyPath))
-            {
-                return null;
-            }
-
-            if (File.Exists(assemblyPath))
-            {
-                var peStream = File.OpenRead(assemblyPath);
-
-                using (var peReader = new PEReader(peStream))
-                {
-                    foreach (var entry in peReader.ReadDebugDirectory())
-                    {
-                        if (entry.Type == DebugDirectoryEntryType.CodeView)
-                        {
-                            var codeViewData = peReader.ReadCodeViewDebugDirectoryData(entry);
-                            var peDirectory = Path.GetDirectoryName(assemblyPath);
-                            return Path.Combine(peDirectory, Path.GetFileName(codeViewData.Path));
-                        }
-                    }
-                }
-            }
-
-            return null;
-        }
-
-        private static bool IsPortable(string pdbPath)
-        {
-            using (var pdbStream = File.OpenRead(pdbPath))
-            {
-                return pdbStream.ReadByte() == 'B' &&
-                    pdbStream.ReadByte() == 'S' &&
-                    pdbStream.ReadByte() == 'J' &&
-                    pdbStream.ReadByte() == 'B';
-            }
-        }
-
         public void Dispose()
```

Comment:
Can this type be removed?

Reply:
I went ahead and just removed the PortablePdbReader.cs